src/Acuminator/Acuminator.Analyzers/Analyzers/Localization/LocalizationMessageHelper.cs

```diff
@@ -11,7 +11,8 @@ namespace Acuminator.Analyzers
 {
     internal class LocalizationMessageHelper
     {
-        private const string _formatRegExp = @"(?<Par>{(\w+|:+)})";
+        private const string _formatRegexString = @"(?<Par>{(\w+|:+)})";
+        private static readonly Regex _formatRegex = new Regex(_formatRegexString, RegexOptions.CultureInvariant);
```

Comment:
@metallmonster there is no `RegexOptions.Compiled` option

Reply:
@SENya1990 as far as I know, this option is not supported in .NET Standard

Reply:
@metallmonster didn't know that. Then everything is ok